src/video-client.js

```diff
@@ -22,7 +24,7 @@ export function openVideo(socket, myPeer) {
         call.answer(stream);
 
         // other user video stream (user A video to user B screen)
-        const [video, videoContainer] = createVideoContainer(false)
+        const [video, videoContainer] = createVideoContainer(false, -1)
```

Comment:
This should really pass the socket.id of whoever connect, but I'm not sure how to get that

Reply:
Current user's socket is in socket.id

If you need to pass another users socket it would be in the payload of the event

Reply:
Oh right the event params aren't in scope.

there is (every connected user) state stored in the peer object in this file too

Reply:
If we could get the peerId from the call and then translate that to the socketID that would be ideal, if we can't get it working we could easily hide it in the demo as well

Reply:
i agree this is definitely good enough

Reply:
A hacky way could be passing the caller Id to server and letting the server pass it to receiver...

Reply:
And that's why it's called a hackathon